packages/components/organisms/f-checkout/src/demo/get-basket-dinein.json

```diff
@@ -0,0 +1,12 @@
+{
+    "BasketId": "11111111",
+    "ServiceType": "Dinein",
```

Comment:
Is this _definitely_ the expected capitalization for this service type from the backend? The contract I've seen specifies "dineIn", but that may be for the menu only rather than for the basket service.

Reply:
It's coming back as `"DineIn"` currently when testing in staging so that's probably what we want to set it to here.